cloud/amazon/ec2_elb_lb.py

```diff
@@ -360,6 +360,29 @@
 except ImportError:
     HAS_BOTO = False
 
+import time
+import random
+
+def _throttleable_operation(max_retries):
```

Comment:
Agree :) See my comment at the end of #2968 where this sort of thing was also discussed.